deployments/kubernetes/install/helm/istio-cni/templates/istio-cni.yaml

```diff
@@ -132,6 +131,34 @@ spec:
               name: cni-bin-dir
             - mountPath: /host/etc/cni/net.d
               name: cni-net-dir
+      # This container installs the Istio CNI binaries
+      # and CNI network config file on each node.
+        - name: repair-cni
```

Comment:
As others commented, this should not run on every single node.
A Deployment with `priorityClassName: system-cluster-critical` and a single replica per cluster would be sufficient.

Reply:
I spoke with Romain offline and we resolved this, we believe that the fieldSelector on nodeName will be sufficient to address the performance concerns here.

Reply:
That's good. Thanks for the explanation.